code/modules/networks/computer3/terminal.dm

```diff
@@ -95,6 +96,10 @@ file_save - Save file to local disk."}
 							src.net_number = new_net_number
 
 					src.peripheral_command("subnet[src.net_number]", null, "\ref[src.netcard]")
+				if (command_list.len > 1)
```

Comment:
lentgh

Reply:
Sorry? I don't understand what you want changed.

Reply:
```suggestion
				if (length(command_list))
```

Reply:
Should we also change the one on line 90? That's where I copied that line from.